src/components/Hero/BenchmarkGraphs.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import styled from 'styled-components';
+
+import BenchmarkGraph from './BenchmarkGraph';
+import constants from '../constants';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin: 48px -8px;
+
+  @media screen and (max-width: ${constants.wrapMobile}) {
+    display: block;
+    margin-left: 0;
+    margin-right: 0;
+  }
+`;
+
+const mapData = (data, key) => data.map(item => ({ Percentage: (`${item.percentile * 100}%`), Value: Number(item[key].toFixed(2)) }));
+
+const getData = (data, strings) => [
+  { title: strings.benchmarks_gold_per_min, data: mapData(data, 'gold_per_min'), color: constants.golden },
```

Comment:
I think you can reuse:
https://github.com/odota/web/blob/c39e25ec854f176ab8ee7bb130d98aad378d5e4c/src/lang/en-US.json#L922